File: twosum.py
nums=[2,7,11,15]
target=9
dic={}
def twoSum(nums, target):
	for x in range(len(nums)):
		if str(target-nums[x]) in  dic.keys():
			print ([dic[str(target-nums[x])],x])
		dic[str(nums[x])]=x 
twoSum(nums,target) 

File: twosum.java
import java.util.*;
class Solution {
	public static void main(String args[]){
		Scanner sc=new Scanner(System.in);
	}
    public int[] twoSum(int[] nums, int target) {
        for(int i=0;i<nums.lenght;i++){
            int a=target-nums[i];
            for (int j=nums.length-1;nums[j]>=a;j--){
                if (nums[j]==a){
                    return [i,j];
                }
            }
            return [-1,-1];
        }
    }
}
